b/bolt/bolt_rhel_ubi8.4.sh

```diff
@@ -0,0 +1,64 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : bolt
+# Version       : v1.3.1
+# Source repo   : https://github.com/boltdb/bolt
+# Tested on     : RHEL ubi 8.4
+# Script License: Apache License, Version 2 or later
+# Maintainer    : BulkPackageSearch Automation vikas.gupta8@ibm.com
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_PATH=github.com/boltdb/
+PACKAGE_NAME=bolt
+PACKAGE_VERSION=v1.3.1
+PACKAGE_URL=https://github.com/boltdb/bolt
+
+yum -y update && yum install -y wget make
+
+# Install Go and setup working directory
+wget https://golang.org/dl/go1.7.4.linux-ppc64le.tar.gz && \
+    tar -C /bin -xf go1.7.4.linux-ppc64le.tar.gz && \
+    mkdir -p /home/tester/go/src /home/tester/go/bin /home/tester/go/pkg /home/tester/output
+
+export PATH=$PATH:/bin/go/bin
+export GOPATH=/home/tester/go
+export GOROOT=/bin/go/
+
+mkdir -p output
+
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+
+#export GO111MODULE=on
+
+if ! go get -d -t $PACKAGE_PATH$PACKAGE_NAME; then
```

Comment:
Should the `PACKAGE_VERSION` be used here?

Reply:
Go version 1.7.x does not have feature to get specific version. Go version 1.7.x does not have feature of go.mod hence could not get specific version.
the bolt module(v1.3.1) was last build with version 1.7.x. Now bolt module is deprecated and archived in github.

Reply:
OK, do we know which version gets installed using this command then? We'll need to mention that one in the header, if different.

Reply:
It gets the master branch.

Reply:
Done. Changes the script to use specified version.